src/include/duckdb/execution/index/art/node4.hpp

```diff
@@ -13,10 +13,11 @@ namespace duckdb {
 
 class Node4 : public Node {
 public:
-	Node4(ART &art, size_t compression_length);
-
+	explicit Node4(size_t compression_length);
+	~Node4() override;
 	uint8_t key[4];
-	unique_ptr<Node> child[4];
+	// Pointers to the child nodes
+	uint64_t children[4];
```

Comment:
Can we create a helper class for this, that includes methods and destructors, e.g.:

```cpp
struct SwizzleablePointer {
   ~ SwizzleablePointer();

   int64_t n;

public:
   bool IsSwizzled();
   ...
};
```